model/riscv_sys_regs.sail

```diff
@@ -309,6 +311,9 @@ function haveZfh()     -> bool = (misa[F] == 0b1) & (mstatus[FS] != 0b00)
 /* V extension has to enable both via misa.V as well as mstatus.VS */
 function haveVExt()    -> bool = (misa[V] == 0b1) & (mstatus[VS] != 0b00)
 
+/* Zcb has simple code-size saving instructions. (The Zcb extension depends on the Zca extension.) */
+function haveZcb()     -> bool = sys_enable_zcb()
```

Comment:
Not opposed to the command line flag, but is there a reason this is not a default-true function like most other extensions? (well it still needs to check for C, but otherwise default true).

Reply:
Nope, it was a totally arbitrary choice. Happy to change it!

Although it does make the command line flag more awkward (`-disable-zcb`?). Are there any extensions that can be toggled but are on by default?

Reply:
Internally we use a JSON file for all these settings. Maybe it would be worth upstreaming that because adding these flags is extremely tedious, and using a JSON file makes it slightly less so.

It does require C++ for the JSON parser though (nlohmann's).